crates/context_aware_config/src/api/config/handlers.rs

```diff
@@ -540,6 +538,7 @@ async fn reduce_config(
 async fn get(
     req: HttpRequest,
     db_conn: DbConnection,
+    query_map: Query<QueryMap>,
```

Comment:
Can we take this in body instead of query params?  Take for example this context in MJOS:
```
"in": [
                            {
                                "var": "clientId"
                            },
                            [
                                "frnd",
                                "pocketfm",
                                "stageharyanvi",
                                "stageott",
                                "stagerajasthani",
                                "rapido",
                                "porterdriver",
                                "fancode",
                                "hungerbox",
                                "purplle.com",
                                "myteam11",
                                "htott",
                                "urbanclap",
                                "rebelfoods",
                                "bbinstant",
                                "dishtv",
                                "ballebaazi",
                                "lenskart",
                                "wiom",
                                "physics",
                                "mamaearth",
                                "mamaearthacc",
                                "passiongaming",
                                "playerzpot",
                                "playerzpot1",
                                "vrlbus",
                                "dishd2h",
                                "classicrummy",
                                "cultfit",
                                "cultsport",
                                "curefitsc",
                                "sugarfit",
                                "watcho",
                                "kotakbima",
                                "kotakd2c",
                                "kotakgenie",
                                "kotakgenienew",
                                "kotakinsurance",
                                "kotakparivartan",
                                "spencers",
                                "pocket52",
                                "platos",
                                "bharatx",
                                "sostronk",
                                "myscoot",
                                "ausfbank",
                                "ausfcasa",
                                "milaap",
                                "bestprice",
                                "buystars",
                                "supersixsports",
                                "mewt",
                                "kapiva",
                                "tamasha",
                                "citymall",
                                "udaan",
                                "nuego",
                                "orangehealth",
                                "sidsfarm",
                                "jockeyindia",
                                "magicbrickspp",
                                "bluestone",
                                "gyftr",
                                "dunzob2b",
                                "newme",
                                "marrowmed",
                                "mxplayer",
                                "travclan",
                                "zaaroz",
                                "skillpattipayout",
                                "et",
                                "reingames",
                                "lalpathdigital",
                                "nuawellness",
                                "toothsi",
                                "healthifyme",
                                "cloudkitch",
                                "joshtalks",
                                "mylo",
                                "damensch",
                                "truefan",
                                "supersixvusport",
                                "boldcare",
                                "bestsellerjacknjones",
                                "turftown",
                                "beato",
                                "velocity",
                                "nxtwave",
                                "bestsellerveromoda",
                                "u-next",
                                "unext",
                                "vijaysales",
                                "bestselleronly",
                                "milton",
                                "fortum",
                                "boltearth",
                                "pluckk",
                                "getmeganew",
                                "getmeganewpoker",
                                "upsurge",
                                "laundrymate",
                                "lifecell",
                                "connectandheal",
                                "hwayz",
                                "werize",
                                "tripsygames",
                                "fantasykhiladi",
                                "stockgro",
                                "unipin",
                                "fitpass",
                                "tyke",
                                "bioayurveda",
                                "livemint",
                                "cars24",
                                "houseofx",
                                "mytrident",
                                "orixindia",
                                "getplus",
                                "cybstimezone",
                                "mirraw",
                                "circlechess",
                                "vistarooms",
                                "fastandup",
                                "sixdreams",
                                "condenast",
                                "giggles",
                                "goldpe",
                                "dineout",
                                "nutristar",
                                "lohono",
                                "zodiac",
                                "salarybox",
                                "admireme",
                                "royalmart"
                            ]
                        ]
```
This may exceed the URL limit of 2048 characters, also it's hard to read and debug
This should be a different API too with method POST

Reply:
this is just a refactor
also, in none of the endpoints we are taking jsonlogic object in query, over here and all other places we are taking `?clientId=giggles` format of data only

Reply:
Can we add in the PR description that we are resolving, I forgot about this